google/cloud/storage/internal/grpc/default_options.cc

```diff
@@ -53,6 +54,20 @@ int DefaultGrpcNumChannels(std::string const& endpoint) {
   return (std::max)(kMinimumChannels, static_cast<int>(count));
 }
 
+bool GrpcEnableMetricsIsSafe() {
+#ifndef GRPC_CPP_VERSION_MAJOR
+  return false;
+#else
+  return (
+      GRPC_CPP_VERSION_MAJOR >= 1 &&
+      (                                                                    //
+          (GRPC_CPP_VERSION_MINOR == 63 && GRPC_CPP_VERSION_PATCH > 1) ||  //
+          (GRPC_CPP_VERSION_MINOR == 64 && GRPC_CPP_VERSION_PATCH > 1) ||  //
```

Comment:
I am confused here. I only see tags for `1.63.0`, `1.64.0`.  https://github.com/grpc/grpc/tags

We are assuming that they will add a `1.63.1` that won't fix the problem, then add a `1.63.2` that will fix the problem?

Reply:
My bad, fixed.